packages/forms/src/styled/file-upload/StyledFileUpload.ts

```diff
@@ -77,11 +61,41 @@ const colorStyles = (props: ThemeProps<DefaultTheme> & IStyledFileUploadProps) =
   `;
 };
 
+const sizeStyles = (props: ThemeProps<DefaultTheme> & IStyledFileUploadProps) => {
+  const marginTop = props.theme.space.base * (props.isCompact ? 1 : 2);
+  const paddingHorizontal = props.theme.space.base * (props.isCompact ? 7.5 : 15);
+  const paddingVertical = math(
+    `${props.theme.space.base * (props.isCompact ? 2.5 : 5)} - ${props.theme.borderWidths.sm}`
+  );
+  const fontSize = props.theme.fontSizes.md;
+  const lineHeight = getLineHeight(props.theme.space.base * 5, fontSize);
+
+  return css`
+    padding: ${paddingVertical} ${paddingHorizontal}px;
+    min-width: 4em;
```

Comment:
Any reason not to use theme here?

Reply:
This decision precedes this PR. There is no theming value for `em` units. And when that is the determined usage, it is implicitly themed, because it is tied to an existing `fontSize` theming value.